_data/simple-icons.json

```
@@ -6976,7 +6976,7 @@
         },
         {
             "title": "Lemmy",
-            "hex": "FFFFFF",
+            "hex": "000000",
```

```suggestion
            "hex": "14854F",
```
Might be a better choice in this case. What do you think?

---

Sure.